.github/workflows/functional.yml

```diff
@@ -0,0 +1,18 @@
+name: Functional
```

Comment:
Very cool. Let me know what you need from me to enable this.

Reply:
GitHub Actions will start executing the workflow on future PRs, as soon as this one is merged.